comp/dogstatsd/server/server.go

```diff
@@ -64,6 +66,11 @@ var (
 	tlmChannel            = telemetry.NewHistogramNoOp()
 	defaultChannelBuckets = []float64{100, 250, 500, 1000, 10000}
 	once                  sync.Once
+
+	// defaultSocket will always be opened as long as the parent directory is available,
+	// usually created by systemd. If the user-configured socket differs from this one,
+	// 2 sockets will be listening.
+	defaultSocket = "/var/run/datadog/dsd.socket"
```

Comment:
Would it make sense to make this a constant?

Reply:
I initially had a constant, but switched to a variable to be able to better test it.